src/content/overview/developer-portal/_index.md

```diff
@@ -1,12 +1,33 @@
 ---
 title: Developer portal
-description: The central Giant Swarm self-service portal for your platform engineers and developers.
+description: Our Backstage-based developer portal is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so engineers using your platform find all the information they need in the right place.
```

Comment:
Please avoid our/your/we/you. This description is also too long for readers to take a look, and contains lots of fill words. I recommend extending the previous description a little:

```suggestion
description: The Giant Swarm platform's self-service portal for platform engineers and developers. Based on Backstage.
```